jovian/utils/commit.py

```diff
@@ -122,6 +123,20 @@ def commit(message=None,
 
     is_cli = kwargs.get('is_cli', False)
 
+    # To commit colab notebooks
+    if in_colab():
```

Comment:
can we move this block near kaggle commit?

Reply:
We will need add additional checks to 3 blocks move this near kaggle commit